src/D2L.Bmx/WriteHandler.cs

```diff
@@ -83,15 +89,19 @@ internal class WriteHandler {
 			}
 		}
 
-		// TODO: Replace with call to function to get AWS credentials and write them to credentials file
-		Console.WriteLine( string.Join( '\n',
-			$"Org: {org}",
-			$"Profile: {profile}",
-			$"User: {user}",
-			$"Account: {account}",
-			$"Role: {role}",
-			$"Duration: {duration}",
-			$"nomask: {nomask}" ) );
+		var credentialsFile = new SharedCredentialsFile();
+		if( !string.IsNullOrEmpty( output ) ) {
+			if( !Path.IsPathRooted( output ) ) {
+				output = "./" + output;
+			}
+			credentialsFile = new SharedCredentialsFile( output );
+		}
 
+		var profileOptions = new CredentialProfileOptions {
+			Token = tokens.AwsSessionToken,
+			AccessKey = tokens.AwsAccessKeyId,
+			SecretKey = tokens.AwsSecretAccessKey
+		};
+		credentialsFile.RegisterProfile( new CredentialProfile( profile, profileOptions ) );
```

Comment:
that worked out rather neatly compared to custom parsing / writing to files. neat